CMakeLists.txt

```diff
@@ -78,6 +78,7 @@ endif()
 set(DLR_LINKER_LIBS "")
 
 set(TVM_SRC "${PROJECT_SOURCE_DIR}/3rdparty/tvm")
+set(JSON_SRC "${PROJECT_SOURCE_DIR}/3rdparty/nlohmann")
```

Comment:
`json.hpp` is not compiled to `.o` file but included to other `.cc` files. So, we do not need to add it to `SRC`